agent/ebs/watcher_linux_test.go

```diff
@@ -0,0 +1,21 @@
+//go:build linux && unit
+// +build linux,unit
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ebs
+
+const (
+	CSIDriverSocketPath = "/var/run/ecs/ebs-csi-driver/csi-driver.sock"
```

Comment:
nit: Any reason why we need to make this variable global? Seems like we're just using this within the same package.

Reply:
This constant and the file will be removed. It's not needed anymore.